Assets/MRTK/Services/InputSimulation/InputSimulationService.cs

```diff
@@ -302,6 +317,27 @@ public override void Update()
                 {
                     cameraControl.UpdateTransform(CameraCache.Main.transform, mouseDelta);
                 }
+
+                //Allow for the user to select with left click if we are in the gestures mode via the gaze provider input source
```

Comment:
Why not also support clicking when in articulated hands mode and the gaze cursor is active? This for example happens whenever you start the game and have no hands up (see the gaze cursor dot), or when you have no hands visible and say the word "Select" (or press the one key).

Reason I ask is becase most people will have the hands mode to be articulated, not gestures. So, if this requires the mode to be gestures, many people will not get to use or see it. It would be nice if you could just hit play and move the camera and click on things without needing to bring up the hands with the hand rays.

Reply:
Sounds good!

Reply:
So I'm noticing some actions like dragging around objects don't seem to work out of the box using the gaze provider and the select action. Will be looking more into it tomorrow. 

Also I'm noticing that our HandInteractionExamples.unity scene is really small, as in, the objects are all really tiny and bundled close together. Is there a reason it's like this, as there's a bit of whiplash when trying to pan the camera around in that scene due to how _relatively_ fast you go